src/SprykerEco/Zed/Unzer/Business/Import/Mapper/UnzerPaymentMethodMapper.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\Unzer\Business\Import\Mapper;
+
+use ArrayObject;
+use Generated\Shared\Transfer\PaymentMethodTransfer;
+use Generated\Shared\Transfer\UnzerApiPaymentTypeTransfer;
+use SprykerEco\Zed\Unzer\UnzerConfig;
+
+class UnzerPaymentMethodMapper implements UnzerPaymentMethodMapperInterface
+{
+    /**
+     * @var \SprykerEco\Zed\Unzer\UnzerConfig
+     */
+    protected $unzerConfig;
+
+    /**
+     * @param \SprykerEco\Zed\Unzer\UnzerConfig $unzerConfig
+     */
+    public function __construct(UnzerConfig $unzerConfig)
+    {
+        $this->unzerConfig = $unzerConfig;
+    }
+
+    /**
+     * @param \ArrayObject $unzerApiPaymentTypeTransfers
```

Comment:
Please, specify the doctypes for `\ArrayObject` here and in other new methods.

Reply:
Fixed